roles/add_livemigration_user/tasks/main.yml

```diff
@@ -8,8 +8,8 @@
       name: "{{ livemigration_user }}"
       shell: /bin/bash
       system: true
-      group: libvirt
-      create_home: true
+      groups: qemu,haclient,libvirt
+      create_home: false
```

Comment:
This will not work on Debian and Centos.

Reply:
because of the groups ?

Reply:
Maybe. But it the crate_home no which will be an issue during the ssh key setup.